src/framed/std/core.cljc

```diff
@@ -121,10 +122,11 @@
   ([f y x]
    (f x y)))
 
-(defmacro future-loop
-  "Execute body repeatedly within a future, returning the future"
-  [& body]
-  `(future
-     (loop []
-       ~@body
-       (recur))))
+#?(:clj
+  (defmacro future-loop
```

Comment:
I tried running `future-loop` in cljs and it didn't throw an exception or anything but it didn't work the way it was supposed to, so I went ahead and scoped it to clj only.

And to answer your question Dan, for including macros, it seems that can be done out of the box in a clj or cljc file